crates/provider/src/provider/trait.rs

```diff
@@ -343,6 +343,16 @@ pub trait Provider<T: Transport + Clone = BoxTransport, N: Network = Ethereum>:
         Ok(block)
     }
 
```

Comment:
needs docs:
 /// Returns the number of transactions in a block from a block matching the given block hash.

Reply:
ah, good catch. ty.

Reply:
fixed